### DISCUSSION TITLE
Why are you requesting the /user/me interface {"data":"xxx"}

### DISCUSSION THREAD

**Discussion by zhuyifang** [directus/directus#7186] [Archive]
Why are you requesting the /user/me interface
The only data returned is:
`{"data": "cc967d2d-e5e2-4de5-b1d8-6c93a417c4b5"}`

Is it a problem with the permissions setting of the user group where the token is passed?

I use the following command to request：
`curl --location --request GET 'http://127.0.0.1:4000/users/me?fields=first_name,id,email,avatar, last_name,role' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6ImNjOTY3ZDJkLWU1ZTItNGRlNS1iMWQ4LTZjOTNhNDE3YzRiNSIsImlhdCI6MTYyODA3MDA0MCwiZXhwIjoxNjI4MDcwOTQwfQ.zZDPCjCjfMzmHHbyoGj9d0uIG7IhQHE3e2iPpSsQoHU'`

**Comment by zhuyifang** [comment]
需要在后台启用权限：APP的访问

**Reply by rijkvanzanten** [reply]
你是对的。
